python/cuml/test/test_metrics.py

```diff
@@ -152,3 +153,59 @@ def test_rand_index_score(name, nrows):
     cu_score_using_sk = sk_ars(y, cu_y_pred)
 
     assert array_equal(cu_score, cu_score_using_sk)
+
+
+@pytest.mark.parametrize('datatype', [np.float32, np.float64, np.int])
```

Comment:
why the test for more than int32? The conversion of datatypes is tested by test_input_utils

Reply:
Agreed, that was a mistake.